src/libs/ReportActionsUtils.ts

```diff
@@ -107,63 +95,116 @@ function isCreatedAction(reportAction: OnyxEntry<ReportAction>): boolean {
 function isDeletedAction(reportAction: OnyxEntry<ReportAction | OptimisticIOUReportAction>): boolean {
     const message = reportAction?.message ?? [];
 
+    if (!Array.isArray(message)) {
```

Comment:
Looks like the present check checks for the field `deleted` in `message[0]` but here we are assuming `message` is `undefined` or `null`. Is this change also mentioned anywhere as part of phase 2?

Reply:
I'm wrong we should return it as `message?.html === '' ?? message?.deleted`.